02-express-tutorial/Public/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<p>Hello Everyone.</p>
+<p>My name is Melanie.</p>
+
+<button id="fetchProductsButton">Fetch Products</button>
+    <div id="productDisplay"></div>
+
+
+<script>
+
+document.getElementById('fetchDataButton').addEventListener('click', () => {
```

Comment:
Nitpick (doesn't require change):
When using document selectors, it's always best to check if the selected node is defined before using methods. For example, if you happen to misspell an id, then you'll get an error logged to the console that states `can't read property "addEventListener" of undefined`.